superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -581,6 +581,9 @@ function ChartList(props: ChartListProps) {
             'the database configuration are not present in export files, and ' +
             'should be added manually after the import if they are needed.',
         )}
+        confirmOverwriteMessage={t(
+          'One or more charts to be imported already exist.',
```

Comment:
we could make this more modular by making the message live in the component and change the resource name. For example, `One or more {resourceName} to be imported already exist.` then we wouldn't have to pass the message through the props

Reply:
The problem is that we need to use plural here ("charts") and `resourceLabel` is singular, and we can't build the plural programmatically. For example, we could try to do this:

```javascript
t('One or more %ss to be imported already exist.', resourceLabel);
```

But in Italian (eg) the plural of chart — grafico — is "grafici", and it would break.

We could change the constructor to receive `resourceName` and `resourcePluralName`, but I thought it would be better to have components define the error message, since it allows for customization.